### PR TITLE
Add experimental page & add minnow.

### PR DESCRIPTION
**Pull request by MagneFire**
Adds an experimental page that consists of `beluga`, `minnow` and `tetra`.

This essentially brings back tetra (https://github.com/AsteroidOS/asteroidos.org/commit/892bc90010fbbdcb296828345316df048f0a36da) as support has improved by means of fixing the battery life issue.

I've moved `beluga` to experimental since it suffers from weird graphical glitches that probably originate from the framebuffer driver.

It also adds `minnow` but I have to ~~verify~~ the support matrix, ~~hence the draft for the moment.~~ Matrix has been updated and USB connection related message is now included.

Previews:
Screenshot from 2022-07-23 23-22-26

Screenshot from 2022-07-24 21-56-48

### CODE REVIEW

**Comment by jrtberlin** [comment]
I think this is a good idea. This way people can still find tetra.

**Comment by docgalaxyblock** [comment]
You forgot to update the `porting-status` regarding `minnow` and maybe rename the page & title to `Moto360 2014 (minnow) `

**Comment by MagneFire** [comment]
> You forgot to update the `porting-status` regarding `minnow` and maybe rename the page & title to `Moto360 2014 (minnow) `

Thanks for the feedback :+1: 

I've updated the `porting-status`. I did consider changing the title too, but it's not referred to the 2014 variant. Instead just `Moto 360` or `Moto  360 Gen 1`. (https://forum.xda-developers.com/c/smartwatches.2102/ and https://docs.google.com/spreadsheets/d/1lbKhNnFBARrT5TVo3bkyp1UzT0S5Iahy3W9dAQf_sfc/view)

**Review by eLtMosen — COMMENTED** [review]
Very good initiative to make the experimental support watches better discoverable!

File: pages/main/install.md [hunk 1]
```diff
@@ -103,6 +98,23 @@ If you have questions regarding the installation process, please check out the *
   <i>Support: <span class="star-good"><i class="icon ion-md-star"></i><i class="icon ion-md-star"></i></span><span class="star-bad"><i class="icon ion-md-star"></i><i class="icon ion-md-star"></i><i class="icon ion-md-star"></i></span></i>
 </div></a>
 
+<div class="page-header">
+  <h1>Experimental</h1>
+</div>
+
+Some watches are deemed experimental for various reasons. They are not recommended for daily use but is enough to get an overall overview of AsteroidOS:
```
[the remark this answers is not in the document]
Comment (MagneFire) [reply pages/main/install.md]: Thanks! This indeed gives a better impression of what is meant by experimental.

File: pages/wiki/porting-status.md [hunk 2]
```diff
@@ -21,6 +21,7 @@ The WearOS smartwatches are the most widespread and easy to support. The source
 - LG G Watch R *(maintained by atx)*
 - LG Watch Urbane *(maintained by TheAppleMan & kido)*
 - LG Watch W7 *(maintained by dodoradio & MagneFire)*
+- Moto 360 (1nd generation) *(maintained by MagneFire)*
```
Comment (docgalaxyblock) [inline pages/wiki/porting-status.md:24]: typo: `1nd` != first

Comment (MagneFire) [reply pages/wiki/porting-status.md]: Great catch! No idea how I missed this :laughing:

File: pages/main/install.md [hunk 3]
```diff
@@ -103,6 +98,28 @@ If you have questions regarding the installation process, please check out the *
   <i>Support: <span class="star-good"><i class="icon ion-md-star"></i><i class="icon ion-md-star"></i></span><span class="star-bad"><i class="icon ion-md-star"></i><i class="icon ion-md-star"></i><i class="icon ion-md-star"></i></span></i>
 </div></a>
 
+<div class="page-header">
+  <h1>Experimental</h1>
+</div>
+
+The AsteroidOS support level for some watches is currently deemed experimental for various reasons. These watches are not recommended for daily use. Despite yet to be fixed major problems, they might still be able to give an overall impression of AsteroidOS.
```
Comment (FlorentRevest) [inline pages/main/install.md:105]: "Despite yet to be fixed major problems"

Sounds like an odd sentence ?

**Comment by FlorentRevest** [comment]
It's ready to merge but there's just one sentence which looks really weird, you may want to reword it and simplify it and then you can merge it

File: pages/main/install.md (hunk 3 above)
Comment (MagneFire) [reply pages/main/install.md:105]: May as well drop it?

```
They are not recommended for daily use but should still give an overall impression of AsteroidOS.
```

Comment (eLtMosen) [reply pages/main/install.md:105]: Agreed

Comment (FlorentRevest) [reply pages/main/install.md:105]: sgtm

**Pull request merged**